dev/required.opam

```diff
@@ -8,5 +8,6 @@ bug-reports: "n/a"
 synopsis: "OCaml dependencies needed in CI"
 
 depends: [
-  "ocamlformat" {= "0.26.1" }  # used by the pre-commit hook
+  # commented out since it doesn't support 5.2
```

Comment:
we probably need to move it to optional.opam maybe? @mjambon maybe knows better.

Reply:
check latest commit, cleaned it up so it only installs if ocaml < 5.0